examples/additional_examples/mixed_energy_force/methane_train.py

```diff
@@ -0,0 +1,30 @@
+import gaptrain as gt
+from autode.wrappers.keywords import GradientKeywords
+gt.GTConfig.n_cores = 10
```

Comment:
Are comparisons between jobs with different numbers of cores still ok?

Reply:
Wall time no, but I don't think(?) I've done that